src/io/flutter/logging/FlutterLog.java

```diff
@@ -50,16 +50,24 @@
       this.value = value;
     }
 
+    @NotNull
     public static Level forValue(int value) {
       final Level[] levels = Level.values();
-      for (int i = 0; i < levels.length; ++i) {
-        if (value >= levels[i].value && ((i >= levels.length - 1) || value < levels[i + 1].value)) {
+
+      for (int i = levels.length - 1; i >= 0; i--) {
+        if (value >= levels[i].value) {
           return levels[i];
         }
       }
-      return null;
+
+      return NONE;
+    }
+
+    public String toDisplayString() {
```

Comment:
Maybe add `@NotNull`

Reply:
Generally useful I agree, but I tend not to use it on methods where there's no logic in the method, or on people who depend on it.

Reply:
Yeap. Got it.